core/src/main/java/com/linecorp/armeria/common/util/EventLoopThreadFactory.java

```diff
@@ -57,7 +67,12 @@ public EventLoopThreadFactory(String threadNamePrefix, boolean daemon) {
      *
      * @param threadNamePrefix the prefix of the names of the threads created by this factory.
      * @param priority the priority of the threads created by this factory.
+     *
+     * @deprecated Use {@link ThreadFactories#newEventLoopThreadFactory(String, boolean)} or
```

Comment:
Can't create a thread factocy whose `priority` is that we specified here if we use `ThreadFactories#newEventLoopThreadFactory(String, boolean)}`. Could you remove this phrase?

Reply:
Removed it.